data/locale/ja.ts

```diff
@@ -9952,4 +9952,4 @@ The LED in the lower right corner of the waveform editor determines whether the
         <translation>出力ゲイン</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
```

Comment:
One of the Japanese translators is working on a Windows machine...

Reply:
I still don't get what's going on here. I haven't found a setting for formatting in my editor (xed). I think only the Swedish translation didn't see this and I haven't touched it until now I think. Are these format fixes correct or should I revert them?